app/v2/controllers/requestParsers/validators/validations/HistoricTaxYearValidation.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.controllers.requestParsers.validators.validations
+
+import v2.models.errors.{MtdError, RuleHistoricTaxYearNotSupportedError, RuleTaxYearRangeInvalidError, TaxYearFormatError}
+
+object HistoricTaxYearValidation {
```

Comment:
Please add a specific `HistoricTaxYearValidationSpec` test class for this (see `TaxYearValidationSpec`) and include tests that passing either max or min value for tax year is ok, but that passing max+1 or min-1 is not ok.